bundle/edu.gemini.gsa.query/src/main/scala/edu/gemini/gsa/query/GsaQuery.scala

```diff
@@ -79,10 +79,16 @@ private [query] object GsaQuery {
       try { s.mkString } finally { s.close() }
     }
 
-    val con = url.openConnection().asInstanceOf[HttpsURLConnection]
+    val con = url.openConnection() match {
+      // Order is important as HttpsURLConnection extends HttpURLConnection
+      case con: HttpsURLConnection =>
+        con.setSSLSocketFactory(GemSslSocketFactory.get)
```

Comment:
The `gemTrustStore.jks` has the root certificates that we need for Let's Encrypt?

Reply:
Yes it contains the root and an intermediate. Should be good until April 2021